tests/test_flows.py

```diff
@@ -383,17 +383,33 @@ def test_channel_selection(self):
         self.check_log(self.repeater1, "agent_wlan2", "CHANNEL_PREFERENCE_QUERY_MESSAGE")
 
         self.debug("Send channel selection request")
-        self.gateway_ucc.dev_send_1905(self.mac_repeater1, 0x8006)
```

Comment:
Commit headline of the first and second commits are identical - please use a different headline (Part 1 and Part 2) can work.

Also there is a typo in the commit message - "teststs" --> "tests"

This can work: `test_flows.py: Enhance channel selection test - part 1`

Reply:
:-1: part 1 part 2 are not useful commit messages. The summary line should say what is done, the body why and more details of what.

So something like:

test_flows.py: channel_selection: add test for empty channel selection request

test_flows.py: channel_selection: add test for power limit

test_flows.py: channel_selection: add test for channel switch

test_flows.py: channel_selection: add test for no channel switch